src/tests.rs

```diff
@@ -417,16 +417,18 @@ pub mod serverlesstest {
     async fn invalid_tx_hash_encoding_test() {
         let app = test::init_service(new_app()).await;
 
-        let invalid_payload = json!({});
+        let payload = json!({
+            "num": 10,
+        });
 
         let req = test::TestRequest::post()
             .uri("/")
             .append_header((
                 "Host",
-                // "0xf17fb991c648e8bdc93f2dcfccc25c98774084ee4ae398f0b289e698b9992303.oyster.run",
+                // "0x9468bb6a8e85ed11e292c8cac0c1539df691c8d8ec62e7dbfa9f1bd7f504e46e.oyster.run",
                 "6F73TEOGJDUL3SJ7FXH4ZQS4TB3UBBHOJLRZR4FSRHTJROMZEMB0.oyster.run",
```

Comment:
should be `SRULW2UOQXWRDYUSZDFMBQKTTX3JDSGY5RROPW72T4N5P5IE4RX0`

Reply:
Done